### DISCUSSION TITLE
Root - OCI permission denied when setting --cpu-rt-runtime=microseconds

### DISCUSSION THREAD

**Discussion by jay-fyi** [containers/podman#10128] [Q&A]
I started the podman using sudo and trying to run the podman with this `cpu-rt-runtime` flag but got the permission denied error.

`sudo podman run -it --rm --entrypoint=bash --cpu-rt-runtime=190000 container-name`

I got the "Error: opening file `cpu.rt_runtime_us` for writing: Permission denied: OCI permission denied"

**Comment by rhatdan** [comment]
When I try this on Fedora 34, I get.

```
# podman run -it --rm --entrypoint=bash --cpu-rt-runtime=190000 alpine
Error: OCI runtime error: realtime runtime not supported on cgroupv2
```

This looks like potentially even root is not allowed to write to this cgroup?  @giuseppe Any ideas?

**Reply by giuseppe** [reply]
that is a hard error from crun, since cgroup v2 doesn't support it.   There is no yet the equivalent of the `cpu.rt_*` files in the cgroup v1 cpu controller.
